src/renderer/components/ft-community-post/ft-community-post.vue

```diff
@@ -62,34 +62,12 @@
     <div
       v-if="type === 'poll'"
     >
-      <div
-        class="poll-count"
-      >
-        {{ postContent.totalVotes }}
-      </div>
-      <div
-        v-for="(poll, index) in postContent.content"
-        :key="index"
-      >
-        <div
-          class="poll-option"
-        >
-          <span
-            class="circle"
-          />
-          <div
-            class="poll-text"
-          >
-            <!-- <img
-              v-if="poll.image != null && poll.image.length >0"
-              :src="getBestQualityImage(poll.image)"
-              class="poll-image"
-              alt=""
-            > -->
-            {{ poll.text }}
-          </div>
-        </div>
-      </div>
+      <ft-community-poll :data="postContent" />
+    </div>
+    <div
+      v-if="type === 'quiz'"
```

Comment:
This can be combined with the check on the line above to be `v-if="type === 'poll' || type === 'quiz'"`, to avoid duplicating it.